governance/program/src/state/proposal.rs

```diff
@@ -622,9 +691,21 @@ impl ProposalV2 {
         vote_threshold: &VoteThreshold,
         vote_kind: &VoteKind,
     ) -> Option<ProposalState> {
-        let min_vote_threshold_weight =
+        let mut min_vote_threshold_weight =
             get_min_vote_threshold_weight(vote_threshold, max_voter_weight).unwrap();
 
+        if let VoteThreshold::AttendanceQuorum(_, pass_level) = vote_threshold {
```

Comment:
For attendance quorum threshold you have to add all approving votes  weight plus deny vote weight with the following logic

If `approve_weight` + `deny_weight` <  `attendance_threshold` then `Proposal` failed 
Evaluate each option individually
if `option_weight` >= `passLevel` and `option_weight` > `deny_weight` then option `Succeeded`